pkg/configs/configs.go

```diff
@@ -66,14 +66,14 @@ func (c RulesConfig) Equal(o RulesConfig) bool {
 // Parse rules from the Cortex configuration.
 //
 // Strongly inspired by `loadGroups` in Prometheus.
-func (c RulesConfig) Parse() ([]rules.Rule, error) {
-	result := []rules.Rule{}
+func (c RulesConfig) Parse() (map[string][]rules.Rule, error) {
+	result := make(map[string][]rules.Rule)
```

Comment:
We generally use literals for construction, rather than `make`

Reply:
👍